ConsentViewController/Classes/MessageWebViewController.swift

```diff
@@ -70,9 +70,17 @@ class MessageWebViewController: GDPRMessageViewController, WKUIDelegate, WKNavig
     }
 
     override func loadView() {
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillHide), name: UIResponder.keyboardWillHideNotification, object: nil)
```

Comment:
@vilasma let's add a comment here explaining we need to add this piece of logic because of an bug on iOS 12/13 regarding the Keyboard shifiting the view up after hiding.